### PR TITLE
Add a test for restarting and recreating compute node.

### PR DESCRIPTION
**Pull request by hlinnaka**
This is working. Let's keep it that way.

### CODE REVIEW

**Comment by hlinnaka** [comment]
> This is working. Let's keep it that way.

I take that back, this is in fact failing :-). I thought it passed on my laptop, but I must've been testing with another branch or something..

**Comment by hlinnaka** [comment]
I added a sleep(5) to the test, to make it pass. See comment in the test for explanation.

File: test_runner/batch_others/test_restart_compute.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    cur.execute("INSERT INTO foo VALUES ('bar2')");
+    cur.execute('SELECT count(*) FROM foo');
+    assert(cur.fetchone()[0] == 2);
+
+    # FIXME: Currently, there is no guarantee that by the time the INSERT commits, the WAL
+    # has been streamed safely to the WAL safekeeper or page server. It is merely stored
+    # on the Postgres instance's local disk. Sleep a little, to give it time to be
+    # streamed. This should be removed, when we have the ability to run the Postgres
+    # instance -> safekeeper streaming in synchronous mode.
+    time.sleep(5)
```
Comment (kelvich) [inline test_runner/batch_others/test_restart_compute.py:47]: Huh, I looked into CLI and it seems that only new_test_master_node() sets proper synchronous_standby_names. IMO we should make this default: https://github.com/zenithdb/zenith/issues/187

File: test_runner/fixtures/zenith_fixtures.py [hunk 2]
```diff
@@ -189,9 +193,19 @@ def config(self, lines):
                 conf.write('\n')
 
     def stop(self):
+        """ stop the server """
         if self.running:
             self.zenith_cli.run(['pg', 'stop', self.branch])
 
+    def destroy(self):
```
Comment (kelvich) [inline test_runner/fixtures/zenith_fixtures.py:200]: JFYI: Few days ago I've added '--destroy' flag to CLI which does exactly this (with intention to enable that as default behavior later)

**Review by kelvich — APPROVED** [review]

File: test_runner/fixtures/zenith_fixtures.py (hunk 2 above)
Comment (hlinnaka) [reply test_runner/fixtures/zenith_fixtures.py]: Ah, cool. I modified the test to use that. Nice to get some test coverage for the '--destroy' flag, too.

**Pull request merged**